content/post/2026/04/april-update-gardens-grass-and-big-fish/index.md

```
@@ -0,0 +1,62 @@
+
+## Touching Grass and Catching Big Fish
+
+Honestly? I've not got much else to blog about because I've been touching grass more. And that's a good thing!
+
+I've been reading more, especially shorter, more personal books. The standout recently has been David Lynch's [*Catching the Big Fish: Meditation, Consciousness, and Creativity*](https://hardcover.app/books/catching-the-big-fish).
+
+It's just a bunch of short paragraphs about his craft, his motivations and the people he's worked with. I was stuck on a train, I'd brought it along with me, train got delayed, finished it in an hour!
+
+Was a nice warm feeling on a morning commute, and it helped me cross off something from my [pile of shame](/garden/books/pile-of-shame/), what beats that?
+
+Plus I'm a big softie and the final page was this: 
```

markdownlint-cli2 (0.22.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)